<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildingTilesDialog</class>
 <widget class="QDialog" name="BuildingTilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Building Tiles</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="overallSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="categorySplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QListWidget" name="categoryList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="categoryListToolbarLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QStackedWidget" name="categoryStack">
          <widget class="QWidget" name="page">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="Tiled::Internal::MixedTilesetView" name="categoryTilesView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="BuildingEditor::FurnitureView" name="furnitureView"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="categoryToolbarLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="addTiles">
         <property name="toolTip">
          <string>Add selected tiles to category</string>
         </property>
         <property name="text">
          <string>&lt;&lt;&lt;</string>
         </property>
         <property name="icon">
          <iconset resource="../tiled.qrc">
           <normaloff>:/images/22x22/add.png</normaloff>:/images/22x22/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeTiles">
         <property name="toolTip">
          <string>Remove selected tiles from category</string>
         </property>
         <property name="text">
          <string>&gt;&gt;&gt;</string>
         </property>
         <property name="icon">
          <iconset resource="../tiled.qrc">
           <normaloff>:/images/22x22/remove.png</normaloff>:/images/22x22/remove.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="tilesetSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QListWidget" name="tilesetList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="tilesetToolbarLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="Tiled::Internal::MixedTilesetView" name="tilesetTilesView">
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionNewCategory">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/document-new.png</normaloff>:/images/16x16/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New Category</string>
   </property>
   <property name="toolTip">
    <string>Add a new category</string>
   </property>
  </action>
  <action name="actionRemoveCategory">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/edit-delete.png</normaloff>:/images/16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove Category</string>
   </property>
   <property name="toolTip">
    <string>Remove the selected category</string>
   </property>
  </action>
  <action name="actionToggleCorners">
   <property name="icon">
    <iconset resource="buildingeditor.qrc">
     <normaloff>:/BuildingEditor/icons/icon_corners.png</normaloff>:/BuildingEditor/icons/icon_corners.png</iconset>
   </property>
   <property name="text">
    <string>Toggle corners</string>
   </property>
   <property name="toolTip">
    <string>Switch between corners and edges</string>
   </property>
  </action>
  <action name="actionMoveCategoryUp">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/go-up.png</normaloff>:/images/16x16/go-up.png</iconset>
   </property>
   <property name="text">
    <string>Move Category Up</string>
   </property>
   <property name="toolTip">
    <string>Move selected category up the list</string>
   </property>
  </action>
  <action name="actionMoveCategoryDown">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/go-down.png</normaloff>:/images/16x16/go-down.png</iconset>
   </property>
   <property name="text">
    <string>Move Category Down</string>
   </property>
   <property name="toolTip">
    <string>Move selected category down the list</string>
   </property>
  </action>
  <action name="actionAddTilesets">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/document-new.png</normaloff>:/images/16x16/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Add Tilesets</string>
   </property>
   <property name="toolTip">
    <string>Add tilesets</string>
   </property>
  </action>
  <action name="actionClearTiles">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/edit-clear.png</normaloff>:/images/16x16/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear Tiles</string>
   </property>
   <property name="toolTip">
    <string>Clear tiles from selected furniture</string>
   </property>
  </action>
  <action name="actionRemoveTileset">
   <property name="icon">
    <iconset resource="../tiled.qrc">
     <normaloff>:/images/16x16/edit-delete.png</normaloff>:/images/16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove Tileset</string>
   </property>
   <property name="toolTip">
    <string>Remove selected tileset</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tiled::Internal::MixedTilesetView</class>
   <extends>QTableWidget</extends>
   <header>BuildingEditor/mixedtilesetview.h</header>
  </customwidget>
  <customwidget>
   <class>BuildingEditor::FurnitureView</class>
   <extends>QTableWidget</extends>
   <header>BuildingEditor/furnitureview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../tiled.qrc"/>
  <include location="buildingeditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BuildingTilesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BuildingTilesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
